content/api/cheatsheet.html

```diff
@@ -1988,19 +1999,19 @@ <h2>Spec (<a href="https://clojure.org/about/spec">rationale</a>, <a href="https
 
 If (check-asserts?) is false at runtime, always returns x. Defaults to
 value of 'clojure.spec.check-asserts' system property, or false if not
-set. You can toggle check-asserts? with (check-asserts bool).">assert</a> <a href="https://clojuredocs.org/clojure_core/clojure.spec/check-asserts" title="clojure.spec.alpha/check-asserts
+set. You can toggle check-asserts? with (check-asserts bool).">assert</a> <a href="https://clojuredocs.org/clojure_core/clojure.spec.alpha/check-asserts" title="clojure.spec.alpha/check-asserts
 ([flag])
   Enable or disable spec asserts that have been compiled
 with '*compile-asserts*' true.  See 'assert'.
 
 Initially set to boolean value of clojure.spec.check-asserts
-system property. Defaults to false.">check-asserts</a> <a href="https://clojuredocs.org/clojure_core/clojure.spec/check-asserts_q" title="clojure.spec.alpha/check-asserts?
+system property. Defaults to false.">check-asserts</a> <a href="https://clojuredocs.org/clojure_core/clojure.spec.alpha/check-asserts_q" title="clojure.spec.alpha/check-asserts?
```

Comment:
link text should be `check-asserts?`

Reply:
Good catch.  I'll see if I can figure out why my code for generating this stuff does that, and fix it.

Reply:
Actually, I think it may be correct, but please double-check me.  `check-asserts` and `check-asserts?` appear one after the other as entries in the sheet, and they both appear to have the correct link text.

Reply:
You can check a different variant of the sheet rendered to HTML here: https://jafingerhut.github.io/cheatsheet/clojuredocs/cheatsheet-tiptip-cdocs-summary.html

It seems to have both check-asserts and check-asserts? with correct links.

Reply:
Oh, I think I just misread the html, nvm